spec/unit/dpkg_database_spec.rb

```diff
@@ -68,6 +68,16 @@
     end
   end
 
+  describe "#determine_changes" do
+    it "checks if a file was deleted although it was reported as changed by dpkg" do
+      expect(system).to receive(:run_command).with("ls", "/etc/sudoers").
+        and_raise(Cheetah::ExecutionFailed.new(nil, nil, nil, nil))
+
+      file, changes, type = subject.determine_changes("??5?????? c /etc/sudoers")
```

Comment:
Useless assignment to variable - `file`. Use `_` or `_file` as a variable name to indicate that it won't be used.<br>Useless assignment to variable - `type`. Use `_` or `_type` as a variable name to indicate that it won't be used.